<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
    <layout class="QVBoxLayout" name="verticalLayout">
      <item>
        <widget class="QPushButton" name="btnLoadMap">
          <property name="text">
            <string>加载地图</string>
          </property>
        </widget>
      </item>
      <item>
        <widget class="QPushButton" name="btnLoadPackages">
          <property name="text">
            <string>加载包裹</string>
          </property>
        </widget>
      </item>
      <item>
        <widget class="QPushButton" name="btnCalculate">
          <property name="text">
            <string>计算路径</string>
          </property>
        </widget>
      </item>
      <item>
        <widget class="QPushButton" name="btnDisplay">
          <property name="text">
            <string>显示结果</string>
          </property>
        </widget>
      </item>
      <item>
        <widget class="QTextEdit" name="txtResult"/>
      </item>
    </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
